.markdownlint.yaml

```diff
@@ -1,11 +1,22 @@
 ---
 default: true
 
-MD013:  # Line Length
-  code_blocks: false
-  tables: false
-  links: false
+# MD013:  # Line Length
+#   code_blocks: false
+#   tables: false
 
-MD033: # Inline HTML
+MD033:  # Inline HTML
   allowed_elements:
-    - br # some tables use line breaks
+    - br  # some tables use line breaks
+    - kbd  # keyboard input
+    - div  # used in ./docs/index.md
+
+# Temporarily disabled until existing violations are fixed so pre-commit PR checks pass
+MD013: false  # Line Length
+MD029: false  # Ordered list item prefix
+MD051: false  # Link fragments should be valid
+MD036: false  # Emphasis used instead of a heading
+MD045: false  # Images should have alternate text (alt text)
```

Comment:
sure feels like we should require this on a public site... I get that going back and adding alttext is a lot to ask...

This feels like one of those situations where "when we add new it should be right" and we should convince someone to go back and make what we have right...

Reply:
(this was only supposed to reflect MD045...)